File: code_challenge_solution/javascript/97 Image viewer.js
```javascript
const images = [
  'image1.jpg',
  'image2.jpg',
  'image3.jpg',
  'image4.jpg',
  'image5.jpg'
];

const viewer = document.getElementById('image-viewer');
const prevBtn = document.getElementById('prev-button');
const nextBtn = document.getElementById('next-button');

let currentIndex = 0;

function showImage() {
  viewer.src = images[currentIndex];
}

function prevImage() {
  currentIndex = (currentIndex - 1 + images.length) % images.length;
  showImage();
}

function nextImage() {
  currentIndex = (currentIndex + 1) % images.length;
  showImage();
}

prevBtn.addEventListener('click', prevImage);
nextBtn.addEventListener('click', nextImage);

showImage();
```

File: code_challenge_solution/c#/97 Image viewer.cs
using System;
using System.Collections.Generic;

namespace ImageViewer
{
    class Program
    {
        static void Main(string[] args)
        {
            List<string> images = new List<string>();

            // Add sample images to the list
            images.Add("image1.jpg");
            images.Add("image2.jpg");
            images.Add("image3.jpg");

            // Display the images
            DisplayImages(images);
        }

        static void DisplayImages(List<string> imageList)
        {
            foreach (string image in imageList)
            {
                Console.WriteLine("Displaying image: " + image);
            }
        }
    }
}